CoreFeatures/Database-postgresql/PostgresSchema/src/main/java/info/smart_tools/smartactors/database_postgresql/postgres_schema/PostgresSchema.java

```diff
@@ -71,13 +72,82 @@ public static void create(final QueryStatement statement, final CollectionName c
             if (options != null) {
-                IndexCreators.writeIndexes(body, collection, options);
+                IndexCreators.writeCreateIndexes(body, collection, options);
             }
         } catch (Exception e) {
             throw new QueryBuildException("Failed to build create body", e);
         }
     }
 
+    /**
+     * Fills the statement body with the ALTER TABLE ADD COLUMN sentence and
+     * CREATE INDEX sentences to add the column and corresponding index for
+     * full text search operation with desired language.
+     * @param statement statement to fill the body
+     * @param collection collection name to use to construct the sequence name
+     * @param options document describing a set of options for the collection creation
+     * @throws QueryBuildException if the statement body cannot be built
+     */
+    public static void addIndexes(final QueryStatement statement, final CollectionName collection, final IObject options)
+            throws QueryBuildException {
+        try {
+            if (options == null) {
+                throw new QueryBuildException("Options for db.collection.addindexes must not be null");
+            }
+            Writer body = statement.getBodyWriter();
+            AlterClauses.writeAddFulltextColumn(body, collection, options);
+            IndexCreators.writeCreateIndexes(body, collection, options);
+        } catch (Exception e) {
+            throw new QueryBuildException("Failed to build create body", e);
+        }
+    }
+
+    /**
+     * Fills the statement body with the ALTER TABLE DROP COLUMN sentence and
+     * DROP INDEX sentences to drop the column and corresponding index for
+     * full text search operation with desired language.
+     * @param statement statement to fill the body
+     * @param collection collection name to use to construct the sequence name
+     * @param options document describing a set of options for the collection creation
+     * @throws QueryBuildException if the statement body cannot be built
+     */
+    public static void dropIndexes(final QueryStatement statement, final CollectionName collection, final IObject options)
+            throws QueryBuildException {
+        try {
+            if (options == null) {
+                throw new QueryBuildException("Options for db.collection.dropindexes must not be null");
+            }
+            Writer body = statement.getBodyWriter();
+            IndexDroppers.writeDropIndexes(body, collection, options);
+            AlterClauses.writeDropFulltextColumn(body, collection, options);
+        } catch (Exception e) {
+            throw new QueryBuildException("Failed to build create body", e);
+        }
+    }
+
+    /**
+     * Fills the statement body with the ALTER TABLE ADD COLUMN sentence and
+     * CREATE INDEX sentences to add the column and corresponding index for
+     * full text search operation with desired language.
```

Comment:
The difference with unsafe version of the statement should be explained here.
```suggestion
     * full text search operation with desired language.
     * This adds an explicit transaction with `BEGIN;` and `COMMIT;` statements to be used in case of autocommit mode.
```

Reply:
done